### PR TITLE
docs: update readme source files

### LINKED ISSUE #50: Add kafka-clients to quickstart installation section in README
Need a way to add `kafka-clients` to README.

```xml
<dependency>	
  <groupId>org.apache.kafka</groupId>	
  <artifactId>kafka-clients</artifactId>	
  <version>2.7.0</version>	
</dependency>
```

### PR DESCRIPTION
**Pull request by anguillanneuf**
Fixes #50 

The region tags should cause autosynth to regenerate the quickstart section of the readme:
- https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/java_library
- https://github.com/googleapis/synthtool/blob/master/synthtool/languages/java.py

### CODE REVIEW

File: .readme-partials.yaml [hunk 1]
```diff
@@ -72,20 +72,14 @@ custom_content: |
...
-  [Google Pub/Sub Lite][product-docs] is designed to provide reliable,
-  many-to-many, asynchronous messaging between applications. Publisher
-  applications can send messages to a topic and other applications can
-  subscribe to that topic to receive the messages. By decoupling senders and
-  receivers, Google Cloud Pub/Sub allows developers to communicate between
-  independently written applications. See the
-  [Google Pub/Sub Lite docs](https://cloud.google.com/pubsub/quickstart-console#before-you-begin)
-  for more details on how to activate Pub/Sub Lite for your project, as well as
-  guidance on how to choose between Cloud Pub/Sub and Pub/Sub Lite.
+  [Google Cloud Pub/Sub Lite][product-docs] is a zonal, real-time messaging
```
Comment (dpcollins-google) [inline .readme-partials.yaml:75]: Remove

File: .readme-partials.yaml [hunk 2]
```diff
@@ -72,20 +72,14 @@ custom_content: |
...
+  service that lets you send and receive messages between independent
+  applications. You can manually configure the throughput and storage capacity
```
Comment (dpcollins-google) [inline .readme-partials.yaml:77]: This reads a bit awkward, just wondering why this is relevant in the intro?

File: .readme-partials.yaml [hunk 3]
```diff
@@ -72,20 +72,14 @@ custom_content: |
...
+  for Pub/Sub Lite systems.
```
Comment (dpcollins-google) [inline .readme-partials.yaml]: I think systems is redundant

File: .readme-partials.yaml (hunk 2 above)
Comment (anguillanneuf) [reply .readme-partials.yaml]: Done.

File: .readme-partials.yaml (hunk 3 above)
Comment (anguillanneuf) [reply .readme-partials.yaml]: https://cloud.google.com/pubsub/lite/docs/

Comment (anguillanneuf) [reply .readme-partials.yaml]: I inherited this from the product page. Let me bring it up with the TW too. In the meanwhile, this might be okay?

Comment (dpcollins-google) [reply .readme-partials.yaml]: This is fine if its just a copy of documentation from elsewhere.

**Review by dpcollins-google — APPROVED** [review]

**Pull request merged**